src/assets/style.css

```diff
@@ -299,3 +344,28 @@ input[type=radio]:checked + .checkbox {
 .wizard--container-wide {
   width: 80%;
 }
+
+.checkbox-container-disabled {
+  pointer-events: none;
+  background-color: #eaeaea;
+}
+
+.wizard--wip {
+  background-color: rgba(92, 234, 165, 0.08);
+  margin: 10px;
+  padding: 10px;
+}
+
+.wizard--container-wip {
+  padding: 50px;
+}
+
+.wizard--wip-input {
+  background-color: #ffffff;
+  margin: 0 10px 10px 10px;
+  font-size: 16px;
+  font-family: Rubik;
+  border-style: none none solid none;
```

Comment:
Unexpected longhand value 'none none solid none' instead of 'none none solid'   shorthand-property-no-redundant-values